src/sdk/namespace_fs.js

```diff
@@ -79,6 +79,46 @@ function sort_entries_by_name(a, b) {
     return 0;
 }
 
+function _get_mtime_from_filename(filename) {
+    if (!filename.includes('_mtime-')) {
```

Comment:
notice there is a gap in this PR, where we kind of ignoring `null` versions. for null versions we also don't have `_mtime` in the filename. 
let's leave it for now and open a gap on the null versions case. we can think if we want to add `mtime` to null versions as well. this will also affect @shirady when implementing suspend versioning

Reply:
Yeah, did not consider that case.
sure, I will make a note of it